synthtool/__main__.py

```diff
@@ -65,14 +64,18 @@ def extra_args() -> List[str]:
 )
 @click.argument("extra_args", nargs=-1)
 def main(synthfile: str, metadata: str, extra_args: Sequence[str]):
-    """Synthesizes source code according to the instructions in synthfile arg.
+    f"""Synthesizes source code according to the instructions in synthfile arg.
 
     Optional environment variables:
       SYNTHTOOL_ARTMAN_VERSION:  The version of artman to use.
       SYNTHTOOL_GOOGLEAPIS:      Path to local clone of https://github.com/googleapis/googleapis
       SYNTHTOOL_GENERATOR:       Path to local gapic-generator directory to use for generation.
                 By default, the latest version of gapic-generator will be used.
       AUTOSYNTH_USE_SSH:         Access github repos via ssh instead of https.
+      {git.PRECLONE_MAP_ENVIRONMENT_VARIABLE}:  Path to a json file.
```

Comment:
this seems like a useful feature, are you thinking that the server that runs `autosynth` will be stateful, and have the repos cloned already? or is this more for development machines.

Reply:
This is purely for autosynth.